benefits/templates/admin/login.html

```diff
@@ -0,0 +1,28 @@
+{% extends "admin/login.html" %}
+{% load i18n static %}
+
+{% block dark-mode-vars %}
+{% endblock dark-mode-vars %}
```

Comment:
This _removes_ the dark mode CSS styles, effectively disabling dark mode on Admin. cc @thekaveman @angela-tran @lalver1 

Right now, the Bootstrap body class declarations are overriding Django's, and we can't really change that. The extrastyle block comes _after_ the style files from Django, and CSS reads the last file's overrides. So the dark mode looks messed up - the text is all dark (as set by Bootstrap). If we could easily set Django's CSS files to be declared _last_ and Bootstrap's first, then we could implement dark mode. But for now it's kinda complex so I think we should just soft launch this site w/o dark mode.